hasher-matcher-actioner/hmalib/hashing/unified_hasher.py

```diff
@@ -72,7 +71,7 @@ def supports(self, content_type: t.Type[ContentType]) -> bool:
         return content_type in self.supported_content_types
 
     def get_hashes(
-        self, content_id: str, content_type: t.Type[ContentType], bytes_: bytes
```

Comment:
lol at us just handing the value right back. 
I might be nice to have if we wanted to try/catch for hashing errors and log them.